lib/board.rb

```diff
@@ -0,0 +1,52 @@
+class Board
+
+	attr_reader :cells
+
+	def initialize
+		self.reset!
+	end
+
+	def cells=(cells)
+		@cells = cells
```

Comment:
Since the behavior you are defining here is the default behavior for the `cells=` method, you could just change your `attr_reader` to `attr_accessor`.